src/events/eventActions.js

```diff
@@ -85,8 +85,9 @@ export const getEvents = (auth) =>
     let lastEventId = data.last_event_id;
 
     dispatch({
-      type: REALM_SET_STREAMS,
+      type: REALM_SET_METADATA,
       subscriptions: data.subscriptions,
+      twenty_four_hour_time: data.twenty_four_hour_time,
```

Comment:
We use camelCasing for JS names, regardless of how it comes from the API.

`twentyFourHourIime: data.twenty_four_hour_time,`